### PR TITLE
PMM-12932 What's new section

### PR DESCRIPTION
**Pull request by matejkubinec**
Adds release notes section when an update is available.

PMM-12932

Link to the Feature Build: SUBMODULES-3717

### CODE REVIEW

**Review by YashSartanpara1 — APPROVED** [review]

File: ui/src/pages/updates/changelog/release-notes/ReleaseNotes.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            h4: {
+              backgroundColor: '#448aff1a',
+            },
+          },
+
+          '&.danger': {
+            borderColor: '#ff1744',
+
+            h4: {
+              backgroundColor: '#ff17441a',
```
Comment (doracretu3pillar) [inline ui/src/pages/updates/changelog/release-notes/ReleaseNotes.tsx:64]: I'm not 100% sure, but without 'ff' is the same color

File: ui/src/types/updates.types.ts [hunk 2]
```diff
@@ -53,3 +55,16 @@ export enum UpdateStatus {
   UpToDate = 'up-to-date',
   UpdateClients = 'update-clients',
 }
+
```
Comment (doracretu3pillar) [inline ui/src/types/updates.types.ts:58]: We will have conflicts with https://github.com/percona/grafana/pull/776/files#diff-336058a35246048f36019d0f034d8e56bfec5bfa5f32be6db5797821c456463a

I've also used the camel case for Changelog everywhere it is used

File: ui/src/types/updates.types.ts [hunk 3]
```diff
@@ -53,3 +55,16 @@ export enum UpdateStatus {
...
+export interface GetChangelogItem {
```
Comment (doracretu3pillar) [inline ui/src/types/updates.types.ts:59]: you can use camel case, I've also renamed the *Changelog* everywhere https://github.com/percona/grafana/pull/776/files#diff-336058a35246048f36019d0f034d8e56bfec5bfa5f32be6db5797821c456463a

File: ui/src/icons/pmm/check.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+export const CheckIcon = (props: SvgIconProps) => (
+  <SvgIcon
+    width="18"
+    height="18"
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="https://www.w3.org/2000/svg"
+    {...props}
+  >
+    <g clip-path="url(#clip0)">
```
Comment (doracretu3pillar) [inline ui/src/icons/pmm/check.tsx:12]: nice approach, will keep that in mind

File: ui/src/pages/updates/changelog/release-notes/ReleaseNotes.tsx (hunk 1 above)
Comment (matejkubinec) [reply ui/src/pages/updates/changelog/release-notes/ReleaseNotes.tsx:64]: image

Shoudln't be the same colors. (at least based on vscode helper)

File: ui/src/icons/pmm/check.tsx (hunk 4 above)
Comment (matejkubinec) [reply ui/src/icons/pmm/check.tsx:12]: Icons were taken from https://github.com/percona/pmm-doc/blob/main/variables.yml

**Review by doracretu3pillar — APPROVED** [review]

**Pull request merged**